build/lib/release.sh

```diff
@@ -284,7 +284,7 @@ function kube::release::create_docker_images_for_server() {
     local images_dir="${RELEASE_IMAGES}/${arch}"
     mkdir -p "${images_dir}"
 
-    local docker_registry="${KUBE_DOCKER_REGISTRY:-gcr.io/google-containers}"
```

Comment:
this line is tricky, because it matters for images which are sideloaded, so both sides need to match. I reverted this because I think the kops-aws job side-loads, using gcr.io/google_containers, but then this broke the gce job (which also side-loads) because I didn't revert its config.

additionally, `anago` explicitly sets this variable, which then may or may not match the side-load config.